deployment/GP03FLMB_Deploy.csv

```diff
@@ -125,3 +125,21 @@ SKQ201920S,,,,GP03FLMB-RIS01-00-SIOENG000,7,1,2019-09-27T21:26:00,,CGMGP-03FLMB-
 SKQ201920S,,,,GP03FLMB-RIS01-03-DOSTAD000,7,1,2019-09-27T21:26:00,,CGMGP-03FLMB-00007,,CGINS-DOSTAD-00519,50.37555,-144.5145,,30,4176,,,
 SKQ201920S,,,,GP03FLMB-RIS01-04-PHSENF000,7,1,2019-09-27T21:26:00,,CGMGP-03FLMB-00007,,CGINS-PHSENF-P0147,50.37555,-144.5145,,30,4176,,,
 SKQ201920S,,,,GP03FLMB-RIS01-05-FLORTD000,7,1,2019-09-27T21:26:00,,CGMGP-03FLMB-00007,,CGINS-FLORTD-01298,50.37555,-144.5145,,30,4176,,,
+SKQ202111S,,,,GP03FLMB-RIM01-00-SIOENG000,8,1,2021-07-18T00:00:00,,CGMGP-03FLMB-00008,,CGCON-SIOCON-13283,50.33,-144.39,,1507,4145,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-ADCPSL007,8,1,2021-07-18T00:00:00,,CGMGP-03FLMB-00008,,CGINS-ADCPSL-20502,50.33,-144.39,,500,4145,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOG060,8,1,2021-07-18T00:00:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOG-13624,50.33,-144.39,,30,4145,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOG061,8,1,2021-07-18T00:00:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOG-13421,50.33,-144.39,,40,4145,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOG062,8,1,2021-07-18T00:00:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOG-13427,50.33,-144.39,,60,4145,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOG063,8,1,2021-07-18T00:00:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOG-13426,50.33,-144.39,,90,4145,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOG064,8,1,2021-07-18T00:00:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOG-13425,50.33,-144.39,,130,4145,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOG065,8,1,2021-07-18T00:00:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOG-13422,50.33,-144.39,,180,4145,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOG066,8,1,2021-07-18T00:00:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOG-12190,50.33,-144.39,,250,4145,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOG067,8,1,2021-07-18T00:00:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOG-13431,50.33,-144.39,,350,4145,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOG068,8,1,2021-07-18T00:00:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOG-13429,50.33,-144.39,,500,4145,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOH069,8,1,2021-07-18T00:00:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOH-13631,50.33,-144.39,,750,4145,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOH070,8,1,2021-07-18T00:00:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOH-12226,50.33,-144.39,,1000,4145,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOH071,8,1,2021-07-18T00:00:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOH-13632,50.33,-144.39,,1500,4145,,,
+SKQ202111S,,,,GP03FLMB-RIS01-00-SIOENG000,8,1,2021-07-18T00:00:00,,CGMGP-03FLMB-00008,,CGCON-SIOCON-14883,50.33,-144.39,,30,4145,,,
+SKQ202111S,,,,GP03FLMB-RIS01-03-DOSTAD000,8,1,2021-07-18T00:00:00,,CGMGP-03FLMB-00008,,CGINS-DOSTAD-00378,50.33,-144.39,,30,4145,,,
+SKQ202111S,,,,GP03FLMB-RIS01-04-PHSENF000,8,1,2021-07-18T00:00:00,,CGMGP-03FLMB-00008,,CGINS-PHSENF-P0144,50.33,-144.39,,30,4145,,,
+SKQ202111S,,,,GP03FLMB-RIS01-05-FLORTD000,8,1,2021-07-18T00:00:00,,CGMGP-03FLMB-00008,,CGINS-FLORTD-01324,50.33,-144.39,,30,4145,,,
```

Comment:
SNs agree with config sheet, although one instrument may have been exchanged and not captured.

Reply:
config and deploy csv updated:  swapped out P0144 for P0180

Reply:
Reviewed configuration, looks good. Swap of P0144 for P0180 looks good.

Reply:
Updated D00001 controllers - they were reversed.  MSIOC changed to 12371; SSIOC changed to 12365.